mmv1/templates/terraform/schema_property.erb

```diff
@@ -26,9 +26,6 @@
 <% if property.default_from_api -%>
 	Computed: true,
 	Optional: true,
-	<% if property.schema_config_mode_attr -%>
```

Comment:
Quick way to stop accidental generation going forward. Will need to fully remove probably after/during Go Rewrite